api/views.py

```diff
@@ -0,0 +1,39 @@
+from rest_framework import generics
+
+from TeachingPlatform.schedules.models import Schedule
+from TeachingPlatform.projects.models import Project, ProjectAssignment
+from TeachingPlatform.subjects.models import Subject
+from TeachingPlatform.users.models import Student, Teacher
+
+from .serializers import ProjectSerializer, ProjectAssignmentSerializer, ScheduleSerializer, SubjectSerializer, \
+    StudentSerializer, TeacherSerializer
+
+
+class ProjectView(generics.ListAPIView):
+    queryset = Project.objects.all()
+    serializer_class = ProjectSerializer
+
+
+class ProjectAssignmentView(generics.ListAPIView):
+    queryset = ProjectAssignment.objects.all()
+    serializer_class = ProjectAssignmentSerializer
+
+
+class ScheduleSerializerView(generics.ListAPIView):
```

Comment:
Using the word `Serializer` in the name of an API view doesn't make much sense, as all API views use serializers :wink: I would call it `ScheduleListView`.

This applies to all api view names with the `Serializer` word :smile: